PULL_REQUEST_TEMPLATE

```diff
@@ -32,6 +35,26 @@ https://github.com/orgs/codice/teams
 - [ ] Documentation Updated
 - [ ] Update / Add Unit Tests
 - [ ] Update / Add Integration Tests
+
+#### Notes on Review Process
+For the general case, merging is gated on the following:
+
+* Review by at least two contributors
+* Review by at least two committers
+* Successful CI
+* Successful static analysis (SonarQube/Coverity/etc.)
+* Heroing of change (details vary by ticket)
+
+Certain exceptions may occur. For example, if there is a problem with the CI environment then a manually performed full build and static analysis check may be substituted _at the discretion of the repository maintainers_. Additionally, repository metadata such as this `PULL_REQUEST_TEMPLATE`, the repository `README`, and other non-functional items may be merged without requiring CI and static analysis.
+
+Finally, an abbreviated review process may be in place for forward-port tickets immediately following branching and prior to branch divergence. In that case, approvals, CI, static analysis, and heroing that occur against the PR against the _release branch_ can be considered sufficient for reviewing the PR against the _master branch_.
```

Comment:
❓ Should we bold **abbreviated review process** (or have a header) to make that part easier to find when that review process is in effect?